main.py

```diff
@@ -1,7 +1,34 @@
 import init_django_orm  # noqa: F401
 
+
 from django.db.models import QuerySet
+from db.models import Genre, Actor
 
 
 def main() -> QuerySet:
-    pass
+    genres = ["Western", "Action", "Dramma"]
+    for genre in genres:
+        Genre.objects.create(
+            name=genre
+        )
+    actors = [
+        ("George", "Klooney"), ("Kianu", "Reaves"), ("Scarlett", "Keegan"),
+        ("Will", "Smith"), ("Jaden", "Smith"), ("Scarlett", "Johansson")
+    ]
```

Comment:
It would be better to place each tuple on a new line to improve the code readability and better separate each element within a list.